src/index.ts

```diff
@@ -17,19 +17,35 @@ class ServerlessFargatePlugin {
 
     private compile(): void {
         const service: any = this.serverless.service;
-        const options: IPluginOptions = service.custom.fargate;
+        const options: IClusterOptions[] = service.custom.fargate;
         const stage: string = service.provider ? service.provider.stage : service.stage;
-
-        // we could want more than one cluster in the future potentially per vpc
-        const vpc: VPC = new VPC(stage, options.vpc);
-        const cluster: Cluster = new Cluster(stage, options, vpc);
-
-        // merge all our stuff into resources
-        Object.assign(
-            this.serverless.service.provider.compiledCloudFormationTemplate.Resources,
-            vpc.generate(),
-            cluster.generate()
-        );
+        
+        //No cluster section specified, don't process
+        if (!options || !options.length) return;
```

Comment:
Maybe add some console.log's here for clarity

Reply:
Done